tooling/bundler/src/bundle/linux/util.rs

```diff
@@ -0,0 +1,91 @@
+// Copyright 2019-2021 Tauri Programme within The Commons Conservancy
```

Comment:
Just a small note. We are now in 2022, and this should  be read by our regex properly:
```
// Copyright 2019-2022 Tauri Programme within The Commons Conservancy
```
Right @lucasfernog ?